assets/src/edit-story/elements/video/display.js

```diff
@@ -62,7 +62,13 @@ function VideoDisplay({
   );
   return (
     <Element>
-      <Video poster={resource.poster} style={style} {...videoProps} loop={loop}>
+      <Video
+        id={`video-${id}`}
+        poster={resource.poster}
```

Comment:
Was the video poster missing here as well?

Reply:
No, it was there (look line 65?).

Reply:
:man_facepalming: